students/preactive/Lesson2/series.py

```diff
@@ -13,12 +13,34 @@ def fib_gen():
     while True:
         a, b = b, a + b
         yield a
-f = fib_gen()
-next(f)
-next(f)
-next(f)
-next(f)
-print(next(f))
+
+
+def fib_gen_teacher():
+    a,b = 0,1
+    while True:
+        a, b = b, a + b
+        yield a
+
+def fib_gen_two():
```

Comment:
It's honestly been a while since I did anything with generators (don't have many use cases in my web dev work). That's great that you're looking at this as an alternative approach to the problem, and I believe they'll by using fibonacci to teach generators in the second quarter so you're getting a massive head start if you plan to continue on ;) 

I think you could just do this to yield 0 for the first item in the sequence though (@natasha-aleksandrova maybe you want to chime in here since you guys were looking at this before): 

```
def fib_gen_teacher():
    yield 0
    a,b = 0,1
    while True:
        a, b = b, a + b
        yield a
```

Reply:
Didnt know you could have multiple yields.  thanks!